.golangci.yml

```diff
@@ -1,27 +1,27 @@
 run:
   timeout: 30m
-  go: '1.20'
-  tests: true
+  go: '1.21'
+  tests: false
   skip-dirs:
     - vendor
+    - tests/e2e
```

Comment:
Why e2e test code is skipped.

Reply:
It doesn't sound like a good reason for me.

Reply:
At least you can keep the old linter for e2e and use new one for feature code.

Reply:
test/e2e is coverred.